jolie-cli/src/main/java/jolie/Jolie.java

```diff
@@ -69,8 +71,15 @@ public static void main( String[] args ) {
 
 		try {
 			CommandLineParser commandLineParser = new CommandLineParser( args, Jolie.class.getClassLoader(), false );
-			final Interpreter interpreter =
-				new Interpreter( commandLineParser.getInterpreterConfiguration(), null, Optional.empty() );
+			Interpreter.Configuration config = commandLineParser.getInterpreterConfiguration();
+			Optional< Value > params = Optional.of( Value.create() );
+			if( config.getParametersFilePath().isPresent() ) {
+				File paramsFilepath = config.getParametersFilePath().get();
+				FileReader fileReader = new FileReader( paramsFilepath );
+				JsUtils.parseJsonIntoValue( fileReader, params.get(), true );
+				fileReader.close();
```

Comment:
Use `try( Reader r = Files.newBufferedReader( ... ) ) { ... }`. That will close the reader automatically.